.github/workflows/ci.yml

```diff
@@ -83,6 +80,31 @@ jobs:
       - name: "Test"
         shell: bash
         run: mvn test -B
+
+  test-OpenJDK:
+    name: "GraalVM on ${{ matrix.os }}"
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.experimental }}
```

Comment:
`experimental` isn't part of the matrix for this job, I think it needs to be added, or else this could just hard-code `continue-on-error: true` for now if we want it to be true for both jobs

We should probably also remove `continue-on-error: true` once we start doing releases that depend on this, but I think it's OK to leave in for now while the graal stuff is being iterated on